docs/pages/api/telemetry/index.ts

```diff
@@ -1,44 +1,32 @@
 import { Analytics } from "@segment/analytics-node";
-import { NextRequest, NextResponse } from "next/server";
+import { NextApiRequest, NextApiResponse } from "next";
 
-export const config = {
-  runtime: "edge",
-};
-
-export default async function forwardTelemetry(req: NextRequest) {
+export default async function forwardTelemetry(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
   if (!process.env.SEGMENT_WRITE_KEY) {
     console.error('Missing required environment variable "SEGMENT_WRITE_KEY".');
-    return NextResponse.json(
-      {
-        error: `Server error`,
-      },
-      { status: 500 }
-    );
+    return res.status(500).json({
+      error: `Server error`,
+    });
   }
 
   if (req.method !== "POST") {
-    return NextResponse.json(
-      {
-        error: `Method ${req.method} Not Allowed`,
-      },
-      { status: 405 }
-    );
+    return res.status(405).json({
+      error: `Method ${req.method} Not Allowed`,
+    });
   }
 
   const analytics = new Analytics({
     writeKey: process.env.SEGMENT_WRITE_KEY,
   });
```

Comment:
I wonder if, now that this is an API route, you can initialize this outside the function scope.

Reply:
we can!